docs/source/release/v6.4.0/Workbench/SliceViewer/Bugfixes/33883.rst

```diff
@@ -0,0 +1 @@
+- Using the Log or SymmetricLog10 colorbar normalisation options on masked data is now disabled to prevent an error.
```

Comment:
In light of this also fixing #33310 please can the release note be updated?

Reply:
I believe the error in #33310 was caused by our switch to matplotlib 3.5, and so is not present in v6.3. I will check and make sure

Reply:
I'm mistaken, it is in v6.3. I'll add a release note for it